ui/apps/platform/cypress/integration/vulnerabilities/snoozeWorkflow.test.ts

```diff
@@ -0,0 +1,58 @@
+import withAuth from '../../helpers/basicAuth';
+import { hasFeatureFlag } from '../../helpers/features';
+
+// Note: each test case should be run against both Node and Platform CVE overview pages
+describe('Node and Platform CVEs - Snooze workflow', () => {
+    withAuth();
+
+    before(function () {
+        if (!hasFeatureFlag('ROX_VULN_MGMT_NODE_PLATFORM_CVES')) {
+            this.skip();
+        }
+    });
+
+    it('should restrict the snooze UI controls', () => {
+        // check that users without VulnerabilityManagementApprovals write-access:
+        //   - do not see table menu options
+        //   - do not see bulk snooze actions
+        // check that users with only read access to VulnerabilityManagementApprovals:
+        //   - do not see table menu options
+        //   - do not see bulk snooze actions
+        // check that users with write access to VulnerabilityManagementApprovals:
+        //   - see table menu options
+        //   - see bulk snooze actions
+        // disable the feature flag (NAME TBD) and check that users with write access to VulnerabilityManagementApprovals:
```

Comment:
Because CI for branch and merge has feature flag on versus release has feature flag off (until turned on near release) separate tests with conditional skip if feature flag is on or off seem more reliable than mocking the featureflags request.

Reply:
So this is a special case, in that as I understand it the feature flag will _never_ be turned on by default in a release build. This is going to be a runtime-toggleable flag that users will opt-in to, if desired, which is why I planned to mock the feature flag here if possible so that we can test both states. The BE APIs will not be visible either when this is disabled.

My thinking is that we will keep this disabled in `pkg/features`, keep it enabled in CI, and then mock the disabled case here for the sake of the permission checks so that we get full coverage.